Transport.php

```diff
@@ -72,4 +80,73 @@ public static function exists($transport)
     {
         return in_array(strtolower($transport), self::get());
     }
+
+    /**
+     * Register a new wrapper in the collection
+     *
+     * @param  string   $protocol The registered protocol ws, wss, http...
+     * @param  callable $factory  The factory callable
+     * @return void
+     */
+    public static function registerWrapper($protocol, callable $factory)
+    {
+        if( isset(static::$_wrappers[$protocol]) ) {
```

Comment:
Ok... Maybe we can just trigger a NOTICE or a WARNING here ? Or you think no notification for the developer...